### DISCUSSION TITLE
Adding trackers

### DISCUSSION THREAD

**Discussion by benefactoress** [VueTorrent/VueTorrent#2055] [Ideas]
Hi,
relative to the qB standard WebUI, it's very complex to add more trackers in VT.  In the standard UI, one can click the active torrent, reveal the torrents tab (if it wasn't already active), then right mouse click to add more torrents.
in VT, one must double click the torrent, then click Torrents, then scroll to the bottom of the screen to then click the button to Add Torrents.
A first start would be to move the Add Torrents and Reannounce All buttons to the top of that window.  A eventual fix would be to add the Add Torrents option to the right-mouse click on the main screen for the torrent.
Thoughts?

**Comment by Larsluph** [comment]
I had the issue recently where I had to manage trackers on several torrents. I must admit that it isn't great as of now.

#1031 and #1100 are already opened so you can subscribe to these to be notified when the update has been done.

**Comment by benefactoress** [comment]
@Larsluph I would say that those two tickets are not addressing what I raised.  I'm not talking about batch update of multiple torrents.  I'm talking about the ability to add trackers to a single torrent, as per the right-mouse click functionality available in the stock qB webUI.  (per the attached screenshot).  I shall however defer to your authority here.
![Screenshot 2024-11-20 094636](https://github.com/user-attachments/assets/57766552-d5a2-4710-9fc8-cb623a1f4a26)

**Comment by Larsluph** [comment]
What you're showing are the two buttons you mentioned that can be moved.

The 2 issues are for modifying trackers directly from the dashboard.

From what I understood you're looking for a quicker way to manage trackers for torrents. qBittorrent's API uses the same route for modifying trackers on a single or multiple torrents.

Except from moving buttons at the top, I'm not sure what else can be done on the detailed torrent page.
